.github/workflows/check-media-attachments.yml

```
@@ -0,0 +1,160 @@
+              commentBody += 'No media-related issues found in the HTML files.\n\n';
+            }
+            
+            commentBody += '*This check was automatically performed when HTML files were modified.*';
+            
+            // Check if we already commented on this PR
+            const { data: comments } = await github.rest.issues.listComments({
+              owner,
+              repo,
+              issue_number: number,
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)